### ISSUE TITLE
"/node_modules/ng2-bootstrap/index"' has no exported member 'DropdownModule'

### ISSUE THREAD

**Issue by DinkuMM** [AndyMeps/ng2-color-picker#6]
Getting following error when try to use color picker
"../node_modules/ng2-bootstrap/index"' has no exported member 'DropdownModule
![errorrrr](https://cloud.githubusercontent.com/assets/17500009/24546128/7fbed0e8-1627-11e7-9fa1-9a769a006908.png)
'

**Comment by norato** [comment]
me too

**Comment by beck24** [comment]
ditto :/

**Comment by AndyMeps** [comment]
Will put in a fix in for this but i've not had much time to work on my angular 2 projects lately; they really need a rewrite to be AoT compliant and just.. well.. better. I'm sorry it's taken so long. I'll try to make a concerted effort to get my ng2 projects up to scratch.

Andy

**Comment by beck24** [comment]
I too have unpaid open source stuff going on and it's hard to find time around the rest of life. I completely understand, and I appreciate it!

**Comment by AndyMeps** [comment]
Hey guys, finally figured this out! Try out v1.2.0 and see how you get on. All feedback is welcome.

Will close this in 7 days.

Cheers,

Andy

**Comment by beck24** [comment]
Thanks, will be checking this out either later today or tomorrow - will report back

**Comment by beck24** [comment]
It doesn't drop down like in the image though, and it appears to have a limit of 10 colors before it errors out - is that correct?

**Comment by AndyMeps** [comment]
That doesn't sound good, how does it error out? I'll set it up on my work PC, clean set up and see how it goes. It does require bootstrap css applied to the site, try adding the below to `index.html` if you haven't already; will get back to you in a few hours.

```html
<!-- index.html -->
<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet">
```

**Comment by AndyMeps** [comment]
Seems to be working fine for me:

* Brand new angular 2 project with @angular/cli, `ng new` project
* Added css link (above) to index.html
* `npm i --save ngx-bootstrap ng2-color-picker`,
* Added `ColorPickerModule.forRoot()` to app.module.ts `imports`
* Added `color = '#ccc';` and `availableColors = [ '#336699', '#6699CC', '#CC9966', '#336699', '#6699CC', '#CC9966', '#336699', '#6699CC', '#CC9966', '#336699', '#6699CC', '#CC9966', '#336699', '#6699CC', '#CC9966',];` to `app.component.ts`
* Added `<color-picker
    [(ngModel)]="color"
    [availableColors]="availableColors">
  </color-picker>` to `app.component.html`
* `npm start` => opened browser and it seems to work fine, with 15 colors.

**Comment by beck24** [comment]
ah, I didn't have the bootstrap css added manually, I guess I assumed that came with the component.
I should have screenshotted the error, it was a js error something about native function having 12 arguments.  Then I reduced the number of colors to 11 and I got the same error stating native function having 11 arguments.  Reduced it to 10 colors and it worked.

I'm trying to use it in ionic 3, that shouldn't make a difference but perhaps something else was interfering.
